package.json

```diff
@@ -42,7 +45,7 @@
         "@babel/preset-env",
         {
           "targets": {
-            "node": "12"
+            "node": "10"
```

Comment:
hey there, why change node.js target to 10 instead of 12?

Reply:
removed, was a debug artefact.